site/recipes/recipe_different_processing_against_stream.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Recipe 4. Applying Different Processing Against a Single Stream
```

Comment:
I think this one needs a better title, I was expecting an example of fan-out where the a single stream is fed into different functions, (e.g.

```
   stream = ...
   stream.filter(...);
   stream.map(...);
```

Instead this is an example of pipeline processing, a stream is transformed/filtered into a different stream.